app/models/sunnyside/incentive.rb

```
@@ -0,0 +1,30 @@
+module Sunnyside
+  # An Incentive to be achieved.
+  class Incentive < ActiveRecord::Base
+    self.table_name = "incentives"
+    belongs_to :group
+    has_many :behaviors, class_name: "Sunnyside::Behavior"
+
+    after_create :create_group_incentives
+
+    validates :description, :group, presence: true
+
+    def self.create_membership_incentives(membership)
+      Incentive.where(group: membership.group).each do |incentive|
+        ParticipantIncentive.create(
```

Is it ok if one of these silently fails to be created?